client/app/components/group/_storeList/storeList.spec.js

```diff
@@ -23,13 +23,24 @@ describe("StoreList", () => {
     "longitude": null
   };
 
+  let storeTwo = {
+    "id": 2,
+    "name": "Teststore2",
+    "description": "all the good stuff again",
+    "group": 1,
+    "address": null,
+    "latitude": null,
+    "longitude": null
+  };
+
   describe("Component", () => {
-    let $scope, $ctrl, $httpBackend;
+    let $scope, $ctrl, $httpBackend, CurrentStores;
```

Comment:
In #405 I changed all tests to use injected services from the controller if possible.
In line 67, you can just append a `$ctrl` before CurrentStores and remove the injection here.